blueflood-core/src/main/java/com/rackspacecloud/blueflood/outputs/formats/MetricData.java

```diff
@@ -40,44 +40,14 @@ public String getUnit() {
     }
 
     public String getType() {
```

Comment:
I kept it because it is being returned to caller. The method is being called from BatchedMetricsJSONOutputSerializer.transformRollupData()

I thought I need to keep backwards compatibility so callers don't break if we suddenly stop sending type. Although, I am kinda doubtful anyone will parse the type we send back.